llm/accelerator_none.go

```diff
@@ -0,0 +1,18 @@
+//go:build !rocm && !cuda
+
+package llm
+
+import (
+	"errors"
+)
+
+const (
+	acceleratorName = "none"
+)
+
+var errNoGPU = errors.New("no accelerator found")
```

Comment:
If this code is built with `//go:build !rocm && !cuda``, doesn't that mean that not just no GPU was found, but also that the binary does not support rocm as well as cuda, right?

i hope i'm understanding this correctly ^^ i've never worked with go build tags.

might be helpful to tell the user that, in so that they don't spend forever trying to get an accelerator working, without realizing its actually their ollama build that's missing the funcitonality. :sweat_smile:

Reply:
I changed the wording, but this error basically exists to make sure that `cpu` understands it has no accelerator, but ErrNoGPU is not undefined. I don't actually expect it's truly reachable or ever user visible.